float64[7] JointPos
